src/data/index.ts

```diff
@@ -1,3 +1,27 @@
-export { people } from './people';
-export { cities } from './cities';
-export { provinces } from './provinces';
+// eslint-disable-next-line @typescript-eslint/ban-ts-comment
+// @ts-nocheck
+import people from '/public/data/people.zip.json' assert { type: 'JSON' };
+import cities from '/public/data/cities.zip.json' assert { type: 'JSON' };
+import provinces from '/public/data/provinces.zip.json' assert { type: 'JSON' };
+
+import type { Person, City, Province } from 'types';
+
+const getPeople = () => {
+  return people;
```

Comment:
```suggestion
  return people as Person[];
```

cast here instead of getData so that the types are also available granularly